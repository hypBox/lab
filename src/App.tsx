import React from "react"
import { Switch } from "react-router-dom"
import { BrowserRouter as Router } from "react-router-dom"
import { getRoutes } from "./utility/routeHelper"

function App() {
  return (
    <>
      <h1>This is header</h1>
      <Router>
        <Switch>{getRoutes()}</Switch>
      </Router>
    </>
  )
}

export default App
