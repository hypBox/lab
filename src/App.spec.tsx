import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

describe('App', () => {
  it('renders as expected', () => {

    const tree = renderer
    .create(<App/>)
    .toJSON();

    expect(tree).toMatchSnapshot();

  });
});
