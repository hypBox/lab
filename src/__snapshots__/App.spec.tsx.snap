// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders as expected 1`] = `
Array [
  <h1>
    This is header
  </h1>,
  <h1>
    THis is MyLab
  </h1>,
]
`;
